### ISSUE TITLE
Optional attributes not working with .dumps()

### ISSUE THREAD

**Issue by BrettStatman** [ramonhagenaars/jsons#78]
In `v1.1.0` I am having trouble serializing an object with optional attributes.

```
import jsons

from dataclasses import dataclass
from typing import List, Optional

@dataclass
class Color:
    name: str
    hex: Optional[str]

@dataclass
class Swatch:
    name: str
    colors: List[Color]

blue = Color('blue', '#0000ff')
green = Color('green', '#00ff00')
swatch = Swatch('swatch 1', [blue, green])

print(swatch)
print(jsons.dumps(swatch))
```

Result
```
Swatch(name='swatch 1', colors=[Color(name='blue', hex='hex'), Color(name='green', hex='hex')])
/site-packages/jsons/_common_impl.py:42: UserWarning: Failed to dump attribute "hex" of object of type "__main__.Color". Reason: 'NoneType' object is not callable. Ignoring the attribute. You can suppress warnings like this using jsons.suppress_warnings().
  warnings.warn(msg_, *args, **kwargs)
{"colors": [{}, {}], "name": "swatch 1"}
```

Result when `hex` is not optional
```
Swatch(name='swatch 1', colors=[Color(name='blue', hex='#0000ff'), Color(name='green', hex='#00ff00')])
{"colors": [{"hex": "#0000ff", "name": "blue"}, {"hex": "#00ff00", "name": "green"}], "name": "swatch 1"}
```

**Comment by ramonhagenaars** [comment]
Hi BrettStatman. That seems to be a bug, I'll dive into it.

**Comment by ramonhagenaars** [comment]
This issue has been fixed in [v1.1.1](https://github.com/ramonhagenaars/jsons/releases/tag/v1.1.1).
